eclipse-scout-core/src/dataobject/serialize/IdDoNodeSerializer.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import {Constructor, DoDeserializer, DoNodeSerializer, DoSerializer, DoValueMetaData, Id, IdParser, scout} from '../../index';
+
+export class IdDoNodeSerializer implements DoNodeSerializer<Id<any>> {
+
+  protected static _ID_PARSER: IdParser; // created on first use. Access using IdDoNodeSerializer._getIdParser()
```

Comment:
Alternatively, we could provide a getSingleton() method on the IdParser class itself.

Reply:
Agree. Fixed.